e2e/support/helpers/e2e-metabot-helpers.ts

```diff
@@ -0,0 +1,81 @@
+import type { StaticResponse } from "cypress/types/net-stubbing";
+
+import { popover } from "./e2e-ui-elements-helpers";
+
+export function assertChatVisibility(visiblility: "visible" | "not.visible") {
+  cy.findByTestId("metabot-chat").should(
+    visiblility === "visible" ? "exist" : "not.exist",
+  );
+}
+
+export function openMetabotViaShortcutKey(assertVisibility = true) {
+  if (assertVisibility) {
+    assertChatVisibility("not.visible");
+  }
+
+  cy.get("body").type("{cmd}b");
+
+  if (assertVisibility) {
+    assertChatVisibility("visible");
+  }
+}
+
+export function closeMetabotViaShortcutKey(assertVisibility = true) {
+  if (assertVisibility) {
+    assertChatVisibility("visible");
+  }
+
+  cy.get("body").type("{cmd}b");
+
+  if (assertVisibility) {
+    assertChatVisibility("not.visible");
+  }
+}
+
+export function openMetabotViaNewMenu(assertVisibility = true) {
+  if (assertVisibility) {
+    assertChatVisibility("not.visible");
+  }
+
+  cy.findByLabelText("Navigation bar").within(() => {
+    cy.findByText("New").click();
+  });
+  popover().findByText("Metabot request").click();
```

Comment:
there's a helper for that!

```
newButton("Metabot request");
```

maybe makes this helper moot, WDYT?

Reply:
I think the asserts are still nice, but I'll definitely take advantage of that helper.